packages/yoastseo/package.json

```diff
@@ -28,7 +28,7 @@
     "clean": "rm -rf build",
     "pretest": "grunt get-premium-configuration",
     "test": "jest",
-    "lint": "eslint src",
+    "lint": "eslint src spec --max-warnings 12",
```

Comment:
For now I set the maximum warnings to 12. But it can also be changed later

Reply:
I'm going to lower it to 10, given that we currently have 10 warnings! 👍